pkgs/development/python-modules/jsonpath-ng/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, decorator
+, fetchFromGitHub
+, ply
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "jsonpath-ng";
+  version = "1.5.2";
+
+  src = fetchFromGitHub {
+    owner = "h2non";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1cxjwhx0nj85a3awnl7j6afnk07awzv45qfwxl5jqbbc9cxh5bd6";
+  };
+
+  propagatedBuildInputs = [
+    decorator
+    ply
+    six
+  ];
+
+  # Tests depends on modules out of the OpenStack eco-system and
+  # those modules were completely removed in 2017
```

Comment:
While `oslotest` is specified in `requirements-dev.txt` I don't see it used anywhere in the project.

Looks like a simple pytest case: https://github.com/h2non/jsonpath-ng/blob/652f665e20ffc403c6f6f93a745e599aa44e3909/Makefile#L27

Reply:
It's [used](https://github.com/h2non/jsonpath-ng/blob/master/tests/test_jsonpath_rw_ext.py#L23) by the tests. Excluding those tests allows us to run the rest.

Thanks for putting me on the right track.